src/SSHDebugPS/IConnection.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Microsoft.SSHDebugPS
 {
-    internal interface IConnection
+    public interface IConnection
```

Comment:
>public [](start = 4, length = 6)

Do you know why you changed this?

Reply:
We had an issue with the Xaml not liking a property i was using it on because it wasn't public and i couldn't make it public without making the interface public. Is it bad to have this as public?

Reply:
Sounds reasonable. But I would add a comment then as to why this is public.

---
In reply to: [287534458](https://github.com/microsoft/MIEngine/pull/868#discussion_r287534458) [](ancestors = 287534458)